### PR TITLE
Feature/cbcccm 31 crnacmp form UI

### PR DESCRIPTION
**Pull request by echo-liang**
# Description

This PR includes the following proposed change(s):

- Refactored the CCCM-CRA form to be built from sections
- Added sample formio json files that builds the sections

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

The app was tested by running 'npm run dev'.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by echo-liang — COMMENTED** [review]
Reviewed with Taylor

File: cccm-frontend/index.html [hunk 1]
```diff
@@ -10,77 +10,63 @@
...
-        <li><a href=".">Disclaimer</a></li>
-        <li><a href=".">Privacy</a></li>
-        <li><a href=".">Accessibility</a></li>
-        <li><a href=".">Copyright</a></li>
-        <li><a href=".">Contact Us</a></li>
-      </ul>
+      <!--Footer section-->
+      <div class="footer">
+        <footer class="footer">
+          <div class="container">
```
Comment (TayGov) [inline cccm-frontend/index.html:58]: Comment

Comment (echo-liang) [reply cccm-frontend/index.html:58]: Answered

File: cccm-frontend/package.json [hunk 2]
```diff
@@ -9,6 +9,8 @@
   },
   "dependencies": {
     "@vue/composition-api": "^1.4.6",
+    "axios": "^0.27.0",
```
Comment (echo-liang) [inline cccm-frontend/package.json:12]:
```suggestion
    "axios": "^0.28.0",
```

**Review by TayGov — APPROVED** [review]
👍

**Pull request merged**